viv_utils/__init__.py

```diff
@@ -70,14 +81,15 @@ def getWorkspace(fp, analyze=True, reanalyze=False, verbose=False, should_save=T
     vw.config.getSubConfig('viv').getSubConfig('parsers').getSubConfig('pe')['loadresources'] = True
     vw.config.getSubConfig('viv').getSubConfig('parsers').getSubConfig('pe')['nx'] = True
     if fp.endswith('.viv'):
-        vw.loadWorkspace(fp)
+        loadWorkspaceFromViv(vw, fp, "'%s' is an invalid .viv file. It may have been generated with Python 2 (incompatible with Python 3).")
```

Comment:
i dont like the passing of exception message here. if the message needs to be updated, i'd recommend catching and rethrowing here.

Reply:
```suggestion
        loadWorkspaceFromViv(vw, fp)
```

Reply:
I wanted to provide more details (the tip to delete the file) for the case in which the user provides a file which is not a `.viv` but there is a `.viv` file in that directory. But maybe this is not needed and one single error is ok.

Reply:
that type of warning is most appropriate for some application to render and show to a user. so i think its best for this library to raise a structured error, and let a client program pick what to do.